android/src/main/java/com/swmansion/gesturehandler/core/FlingGestureHandler.kt

```diff
@@ -3,15 +3,63 @@ package com.swmansion.gesturehandler.core
 import android.os.Handler
 import android.os.Looper
 import android.view.MotionEvent
+import android.view.VelocityTracker
+import com.swmansion.gesturehandler.core.GestureHandler.Companion
+import kotlin.math.hypot
+
+class Vector {
+  var x: Double = 0.0
+  var y: Double = 0.0
+  private var uX: Double = 0.0
+  private var uY: Double = 0.0
```

Comment:
We can use more letters 😄 
```suggestion
  private var unitX: Double = 0.0
  private var unitY: Double = 0.0
```

Or we can go even further:
```suggestion
  private var unit: Vector
```
Though, it may be overcomplicating at this point since we only need those for the dot product.

Reply:
I agree that it looks better with "more letters" (at first I wasn't sure what exactly is `uX`), but creating unit vector inside is a bit too much for me 😅